tsconfig.json

```diff
@@ -8,8 +8,15 @@
       "sass": ["sass/"]
     },
     "rootDirs": ["public/"],
-    "typeRoots": ["node_modules/@types", "public/app/types"]
+    "typeRoots": ["node_modules/@types", "public/app/types"],
+    "allowJs": true
```

Comment:
this is needed so that when js files are picked up by jest and ts-jest they are actually processed.

Reply:
Can't see any immediate consequences of this, but maybe we could have tsconfig.jest.json file that takes care of this?